packages/core/src/components/forms/label.tsx

```diff
@@ -42,7 +42,7 @@ export class Label extends React.PureComponent<ILabelProps, {}> {
         return (
             <label {...htmlProps} className={rootClasses}>
                 {text}
-                {helperText && <span className={classNames(Classes.TEXT_MUTED)}>{" " + helperText}</span>}
+                {helperText && <span className={classNames(Classes.TEXT_MUTED)}>{" "}{helperText}</span>}
```

Comment:
while we're at it, can you remove the `classNames()` call? not necessary for one class.

Reply:
@will-stone i went ahead and made this change here. thanks!